deploy/templates/nstemplatetiers/base/ns_dev.yaml

```diff
@@ -12,6 +12,8 @@ objects:
       openshift.io/requester: ${USERNAME}
     labels:
       name: ${USERNAME}-dev
+      modelmesh-enabled: "true"
```

Comment:
Could you please add some comment explaining:
- that it's for RHODS
- what it does

Reply:
Added a comment with explanation